TCGA/examples.py

```diff
@@ -44,16 +44,28 @@ def upload_examples(cases, username=None, password=None):
         case_name = case_folder.name
         if (cases is None and 'test' not in case_name) or (cases is not None and case_name in cases):
             case_folder_item = get_case_folder_item(client, case_name)
-            image_path = case_folder / (f'{case_name}.svs' if case_name != 'test' else 'tcgaextract_rgb.tiff')
-            if image_path.exists():
-                parquet_path = case_folder / f'{case_name}.parquet'
-                if not parquet_path.exists():
-                    print('Generating parquet file of vector data...')
-                    vector = get_case_vector(case_name=case_name)
-                    vector.to_parquet(parquet_path)
-                print(f'Uploading image and parquet file for {case_name}.')
+            if case_name == 'test':
+                image_path = next(case_folder.glob('tcgaextract_rgb.tiff'), None)
+            else:
+                image_path = next(case_folder.glob('*.svs'), None)
+            if image_path is not None and image_path.exists():
+                print(f'Uploading image for {case_name}.')
```

Comment:
We previously wrote something different to match case names; does this behave in the same way, or have I missed something?

Reply:
This looks right. The glob on line 50 is a nice way to grab whatever SVS file is in that directory. Two comments:
1. This strategy doesn't prevent you from putting two SVS files in the directory, and then receiving the "wrong" one. It might be worthwhile to call `next(..., None)` on the glob one more time to eliminate that case.
2. I don't think you need the `next()` call on line 48; you can just say `image_path = case_folder / 'tcgaextract_rgb.tiff'`, and the `image_path.exists()` guard on line 51 should keep you safe.